Index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mobile First Hussen Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="top-bar">
+        <div class="tool-bar">
+          <div class="logo-setting">
+            <div class="logo">
+              <a href="#"><img src="img/My-Logo.png" alt="HY" /></a>
+              <!-- <img src="img/Hu-Photo-2-3.jpeg" alt="HY" width="76px" height="20px"> -->
+            </div>
+          </div>
+          <nav class="dash-setting">
+            <div class="icons">
+              <div class="icon-Union">
+                <img src="img/Union.png" alt="Three Dashs" />
+              </div>
+            </div>
+          </nav>
+        </div>
+      </header>
+      <section class="content-section">
+        <div class="bg-shapes">
+          <div class="text-1">
+            <h2>I’m Hussen Glad to see you!</h2>
+          </div>
+          <div class="text-2">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <!-- <div class="social-media-setting"> -->
+          <!-- <div class="social-media-title"> -->
```

Comment:
[Optional] I see theres a lot of commented out code in your project. Please note that this is considered bad practice as it makes your code unclean and look very messy. I strongly suggest you go through your entire project and remove all commented out code🫡. Comments should only be used to describe the code